testing/botan/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Alexander F Rødseth <xyproto@archlinux.org>
+# Contributor: Angel Velasquez <angvp@archlinux.org>
+# Contributor: Douglas Soares de Andrade <douglas@archlinux.org>
+# Contributor: d'Ronin <daronin@2600.com>
+# Contributor: Hexchain Tong <richard0053@gmail.com>
+# Contributor: Steve Holweg <skytep@gmail.com>
+
+pkgname=botan
+pkgver=2.3.0
+pkgrel=1
+pkgdesc="Crypto library written in C++"
+arch="all"
+url="https://botan.randombit.net/"
+license="BSD"
+depends=""
+makedepends="python"
```

Comment:
There's no package `python`, it should be `python3` or `python2`. Use Legacy Python (python2) only if it doesn't work with python3.